### PR TITLE
SAF-5990: Add claimsSink and claimsSource config, claims query

### PR DESCRIPTION
**Pull request by andrew-maltsev**

### CODE REVIEW

File: packages/safety-suite-api/src/api/claims/claims.ts [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+import {runApi} from '../../runApi';
+import {PageRequest} from './pageRequest';
+import {PageResponse} from './pageResponse';
+
+export enum ClaimStatus {
+  OPEN = 'Open',
+  CLOSED = 'Closed',
+  LITIGATION = 'Ligitation',
+  SUBRO = 'Surbo'
```
Comment (NickVarnado) [inline packages/safety-suite-api/src/api/claims/claims.ts:12]: typo

File: packages/safety-suite-api/src/api/claims/claims.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+ * @param pageRequest - Default claims by page request with filtering and sorting.
+ * @param customerAlias - Customer alias.
+ * @param apiOptions - Optional options for runApi.
+ * @returns - Claim models page response object.
+ */
+export function getPageClaims(
+  pageRequest: PageRequest,
+  customerAlias: string,
+  apiOptions?: ApiOptions
+): Request<ApiResponse<PageResponse<ClaimModel>>> {
```
Comment (NickVarnado) [inline packages/safety-suite-api/src/api/claims/claims.ts:141]: Should this be an array of ClaimModels?

File: packages/safety-suite-api/src/api/claims/claims.ts (hunk 1 above)
Comment (andrew-maltsev) [reply packages/safety-suite-api/src/api/claims/claims.ts]: fixed

File: packages/safety-suite-api/src/api/claims/claims.ts (hunk 2 above)
Comment (andrew-maltsev) [reply packages/safety-suite-api/src/api/claims/claims.ts:141]: sure comes array -> this is just type for generic PageResponse

export type PageResponse<T> = {
  _embedded: PageResponseData<T>;
}

export type PageResponseData<T> = {
  [key: string]: T[];
};

Comment (andrew-maltsev) [reply packages/safety-suite-api/src/api/claims/claims.ts]: sure, fixed 
```
export interface ClaimResponse {
  claims: ClaimModel[];
}
```

**Comment by andrew-maltsev** [comment]
pageRequest.ts moved into api folder. In the future we will reuse this for different services that will use Query Language requests of Spring Data JPA

**Review by NickVarnado — APPROVED** [review]

**Pull request merged**